conf/threat_intel.json

```diff
@@ -0,0 +1,24 @@
+{
+  "dynamodb_table_name": "table_name",
+  "enabled": true,
+  "excluded_iocs": {
+    "ip": {
+      "10.0.0.0/8": {
+        "comment": "RFC1918"
+      }
+    }
+  },
+  "normalized_ioc_types": {
```

Comment:
I named this `normalized_ioc_types` to be more clear, since the values of these types must map to 'normalized' keys.. for instance, a user can't just put any key in here and it will work.. it must be something that is in the normalized types mapping. does that make sense? since this is the case, I'd like to keep the naming convention